lerna.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "independent",
+  "npmClient": "yarn",
+  "conventionalCommits": true,
+  "useWorkspaces": true,
+  "packages": [
+    "packages/*"
+  ],
+  "ignoreChanges": [
+    "**/__fixtures__/**",
+    "**/__tests__/**",
+    "**/test/**",
+    "**/*.md"
```

Comment:
I am a bit thorn about this. We might want changes to readmes to be released to be reflected on npm. @havardh What do you think?

Reply:
Yes we want that. I'm not sure about the `__tests__` or the `__fixtures__` pattern. I did just merge the `__fixtures__` thing so we'll have to live with that for now.

Reply:
The __fixtures__ looks nice next to __snapshots__ 😛